src/EditorFeatures/Core/Wrapping/BinaryExpression/BinaryExpressionCodeActionComputer.cs

```diff
@@ -105,6 +106,55 @@ private ImmutableArray<Edit> GetUnwrapEdits()
 
                 return result.ToImmutableAndFree();
             }
+
+            private string GetSingleIndentation()
```

Comment:
An option might be possible, but the default should stick to the values provided by the indentation service. These are the values most users will expect because most users are (and have always) using the indentation service for wrapping.

Reply:
> but the default should stick to the values provided by the indentation service. 

i'm ok with that.

Reply:
>  These are the values most users will expect because most users are (and have always) using the indentation service for wrapping, since that is the unconfigurable default wrapping behavior.

Note: i really want to point out that i think this is actually not that compelling an argument to me.  First, i absolutely realize that's what the indentation service does. But, in that sense, that's why it's *less* important that hte refactoring do this.  That's because you can def get this 'indentation' behavior pretty trivially.  You simply press 'enter' where you want the indent.  The real benefit of this refactoring is to get you the formatting you want, which would normally take *more* effort. In that sense, the individual refactorings are more valuable the more work they save you.

So, as before, i think the best result here is simply to have both styles (like with params/args).  This gives hte fast and easy access for people who want to just have 'indent-style' wrapping, while also ensuring that people want 'align-style' wrapping can do that as well.

Reply:
I'll file an issue to consider additional options, but I have no intent to add the functionality in this small big fix PR.

Reply:
➡️ Moved to #32830